src/store/conversationsStore.js

```diff
@@ -691,6 +691,9 @@ const actions = {
 			const response = await fetchConversation(token)
 			dispatch('updateTalkVersionHash', response)
 			dispatch('addConversation', response.data.ocs.data)
+			if (!response.data.ocs.data.callFlag) {
+				dispatch('purgeSpeakingStore', { token })
+			}
```

Comment:
Thanks for a link, will try to replicate this. But the other problem could arise here: 
- We want to store elapsed time as long as call is active (in case a person will leave it for a while, or will try to reconnect). The better way to do it - is to follow a callFlag
- Though we're stil missing the elapsed time for the time we're not present in call, and timers won't survive a page reload - so maybe there's no need to keep it on call leaving? cc @jancborchardt

Reply:
My bad, I was not aware that the speaking time had to survive the participant being in the call; I just assumed that once a participant left the call the speaking time was reset. In that case you are right, that logic would outlive the `SignalingSpeakingHandler` and would need to be done in the store based on the call flag of the conversation.

Reply:
The time would be nice to be preserved across reconnects (in case there is an issue). Is it easier and/or possible as a middle ground to preserve it as long as at least 1 person is in the call, similar like we do for author colors in Text?

Reply:
As was discussed with Jan, we're not going to keep time between call reconnections for now